### PR TITLE
Fixed issue #11 and Validation Example code.

### PR DESCRIPTION
**Pull request by yashkhatri**

### CODE REVIEW

File: src/main/java/org/eclipse/lyo/validation/impl/ValidatorImpl.java [hunk 1]
```diff
@@ -47,176 +43,171 @@
...
-            log.debug("setting uri");
-            resourceModel.setURI(model.getSeq(resource).toString());
-        } catch (Exception e) {
-            log.debug("uri doesn't exist");
-            resourceModel.setURI("No uri exists");
-        }
-
-        resourceModel.setResult(validationResult);
-    }
+	private static final Option<String> OPTION_NONE = Option.apply(null);
```
Comment (berezovskyi) [inline src/main/java/org/eclipse/lyo/validation/impl/ValidatorImpl.java]: we use spaces in Lyo projects

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Lyo Validation library
 
-The library is intended to check whether resource instances conform to SHACL shapes.
+The library validates RDF resources against constraints defined either as [`SHACL`](https://www.w3.org/TR/shacl/) or [`OSLC Resource Shapes`](http://docs.oasis-open.org/oslc-core/oslc-core/v3.0/cs01/part6-resource-shape/oslc-core-v3.0-cs01-part6-resource-shape.html).
```
Comment (berezovskyi) [inline README.md:3]: this code was already in Jad's patch?!

Comment (berezovskyi) [reply README.md:3]: plus the `code` backticks are not really necessary

File: src/main/java/org/eclipse/lyo/validation/impl/ValidatorImpl.java [hunk 3]
```diff
@@ -47,176 +43,171 @@
 import org.eclipse.lyo.validation.shacl.ShaclShapeFactory;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+
+import es.weso.rdf.PrefixMap;
```
Comment (berezovskyi) [inline src/main/java/org/eclipse/lyo/validation/impl/ValidatorImpl.java]: I don't see where did you use these newly added imports

File: src/main/java/org/eclipse/lyo/validation/shacl/ShaclShapeFactory.java [hunk 4]
```diff
@@ -106,6 +106,8 @@ private static ShaclShape createShaclShape(final Class<?> resourceClass,
         // annotations.
         if (!shaclShape.isReadShaclAnnotations()) {
             shaclShape.setShaclProperties(null);
+            shaclShape.setTargetClass(new URI(resourceClass.getAnnotation(OslcNamespace.class).value()
```
Comment (berezovskyi) [inline src/main/java/org/eclipse/lyo/validation/shacl/ShaclShapeFactory.java:109]: how come no unit tests had to be changed?

**Review by berezovskyi — CHANGES_REQUESTED** [review]
Please see the comments attached.

And do you suggest we abanon #10 then?

**Comment by berezovskyi** [comment]
Yash, your GMail email address also does not have an ECA, you will need to fix the email address in your commit and Signed-Off-By line.

**Comment by yashkhatri** [comment]
Yes, this pull request contains the changes from the pull request #10. So we can abandon that.

File: src/main/java/org/eclipse/lyo/validation/shacl/ShaclShapeFactory.java (hunk 4 above)
Comment (yashkhatri) [reply src/main/java/org/eclipse/lyo/validation/shacl/ShaclShapeFactory.java:109]: There wasn't a unit test covering this. That's why this bug was left out. Now, we have an example in ValidationExample.java covering this.

File: README.md (hunk 2 above)
Comment (yashkhatri) [reply README.md:3]: Yes, it's from Jad's patch. 
@jadelkhoury

**Review by berezovskyi — APPROVED** [review]

**Review by berezovskyi — CHANGES_REQUESTED** [review]
The POM file needs to say that the lyo-validator version is 2.3.0-SNAPSHOT, because 2.3.0.M3 has already been released and can't be deployed twice (as it will by the Jenkins build).

**Comment by yashkhatri** [comment]
I think the project name should be updated. It is currently called lyo-validation. Suggestion: 
"org.eclipse.lyo.validation" ?

**Review by berezovskyi — APPROVED** [review]

**Pull request merged**